### PR TITLE
[#165] Minor layout changes

### PR DESCRIPTION
**Pull request by mauriel133**
Changed 'Keywords' for 'Subject' and moved the section to below abstract.
Improved missing fields notification behavior.

TODO:
- [ ]  Implement missing fields notification behavior for non generic resource types.

### CODE REVIEW

File: hs_core/templates/pages/genericresource.html [hunk 1]
```diff
@@ -196,6 +196,54 @@
...
+                    <div class="tags">
+                        <ul id="list-keywords" class="tag-list custom-well">
+                        </ul>
+                    </div>
+                </div>
+                {% endif %}
+            {% else %}
+                <div class="content-block">
+                    <h3>Subject</h3>
+                    <form action="/hsapi/_internal/{{ cm.short_id }}/subject/add-metadata/" id="id-subject"
```
Comment (ironfroggy) [inline hs_core/templates/pages/genericresource.html]: URLs should be constructed with the `{% url %}` tag.

**Comment by mauriel133** [comment]
@mjstealey, I see this branch has a change on django_irods. Is this going to create a problem I need to address before this code is merged?

**Comment by mjstealey** [comment]
@mauriel133 - Unless you made a change to the submodule reference it should not be part of your pull-request.

In this particular case neither of the SHAs listed are the most recent, so yes, please either update the reference to be the most recent for the django_irods submodule, or remove from you pull-request.

**Comment by mauriel133** [comment]
Alright, it has been taken care of.

**Pull request merged**